### PR TITLE
PowerShell Arrays Module - closes #179

### LINKED ISSUE #179: PowerShell Arrays Module
For arrays, how to:
- Create
- Add Elements
- Access Elements
- Update Elements
- Remove Elements

Suggested reference: https://ss64.com/ps/syntax-arrays.html

### PR DESCRIPTION
**Pull request by bob-crutchley**

### CODE REVIEW

File: topics/powershell/modules/arrays/README.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+$myArray = @(1,2,3)
+$myArray[0..2]
+```
+A range and specific indexes can be access by adding `+`:
+```powershell
+$myArray = @(1,2,3,4,5,6,7,8)
+# access the 1st and 3rd elements, also access elements at index 4 to 7
+$myArray[0,2+4..7]
+```
+### Last Element
```
Comment (Matt25969) [inline topics/powershell/modules/arrays/README.md]: An better explanation required here - can this be expanded to include  $myArray[-5] for instance?
Either way it should be mentioned here.

File: topics/powershell/modules/arrays/README.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    $columns = $line -split ","
+    $rows += , $columns
+}
+```
+Notice the way that `$columns` is being appended to `$rows`; a comma has been included to say explicity that we want to append the `columns` variable to `$rows` as an array:
+```powershell
+$rows += , $columns
+```
+#### Iterate the Multi Dimentional Array
+Now the information is sotred in a multi dimentional array, a loop can be used to iterate through it and print out the values in a basic format for a table:
```
Comment (Matt25969) [inline topics/powershell/modules/arrays/README.md]:
```suggestion
Now the information is sorted in a multi dimensional array, a loop can be used to iterate through it and print out the values in a basic format for a table:
```

File: topics/powershell/modules/arrays/README.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+```powershell
```
Comment (Matt25969) [inline topics/powershell/modules/arrays/README.md]: Get that bloody spell checker sorted - I dread to think of the spelling issues we will get over time!

File: topics/powershell/modules/arrays/README.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+Name | Score |
+==============
+Bob | 80% |
+==============
+Jay | 85% |
+==============
+John | 80% |
+==============
+Tadas | 85% |
+```
```
Comment (Matt25969) [inline topics/powershell/modules/arrays/README.md]: ==============

Comment (Matt25969) [inline topics/powershell/modules/arrays/README.md]: Matt | 1000% |

Comment (Matt25969) [inline topics/powershell/modules/arrays/README.md]: I fixed it

File: topics/powershell/modules/arrays/README.md (hunk 1 above)
Comment (bob-crutchley) [reply topics/powershell/modules/arrays/README.md]: I've adding something that hopefully addresses this now.

**Pull request merged**